Dockerfile

```diff
@@ -36,6 +36,13 @@ LABEL io.k8s.display-name="internal-services"
 LABEL io.openshift.tags="internal-services"
 LABEL summary="Konflux Internal Services"
 LABEL com.redhat.component="internal-services"
+LABEL org.opencontainers.image.created="2025-11-24"
```

Comment:
wait, this one should definitely be added by the build service, no? this definitely shouldn't be hardcoded like this I'd say.